NF.jsonld

```diff
@@ -15563,6 +15609,295 @@
+            "rdfs:label": "FundingAgency",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:DccProperty"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "fundingAgency",
+            "sms:required": "sms:false",
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:CreatedOn",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "Refers to when the resource was created on the platform.",
+            "rdfs:label": "CreatedOn",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:SynapseProperty"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "createdOn",
+            "sms:required": "sms:false",
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:CreatedBy",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "Refers to the user who created the resource on the platform.",
+            "rdfs:label": "CreatedBy",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:SynapseProperty"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "createdBy",
+            "sms:required": "sms:false",
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:ModifiedBy",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "Refers to a user who last modified the resource on the platform.",
+            "rdfs:label": "ModifiedBy",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:SynapseProperty"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "modifiedBy",
+            "sms:required": "sms:false",
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:Id",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "The entity id for the resource automatically assigned by the platform.",
+            "rdfs:label": "Id",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:SynapseProperty"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "id",
+            "sms:required": "sms:false",
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:Etag",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "A UUID assigned by the platform.",
```

Comment:
```suggestion
            "rdfs:comment": "Synapse employs an Optimistic Concurrency Control (OCC) scheme to handle concurrent updates. The E-Tag changes every time an entity is updated it is used to detect when a client's current representation of an entity is out-of-date.",
```

I thought this one could use some additional information, so I grabbed and modified the definition from our REST docs https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/Entity.html